internal/cmdrpkgcopy/command.go

```diff
@@ -154,7 +155,7 @@ func (r *runner) getPackageRevisionSpec() (*porchapi.PackageRevisionSpec, error)
 		Revision:       r.revision,
 		RepositoryName: packageRevision.Spec.RepositoryName,
 	}
-	spec.Tasks = []porchapi.Task{{Type: porchapi.TaskTypeEdit, Edit: &r.copy}}
+	spec.Tasks = packageRevision.Spec.Tasks
```

Comment:
Do you happen to know why we end up creating `restClient` in this method instead of using the `client` in the runner ?

Reply:
I think I suggested using client-go instead of the kubectl builder in https://github.com/GoogleContainerTools/kpt/pull/3130#discussion_r871725895. Agree that using the controller-runtime client would be better. We can clean that up at some point.

Reply:
Yes, can be cleaned up later.

Reply:
Replaced the `restClient` with the controller-runtime client.

Reply:
Sweet.